### ISSUE TITLE
Istioctl segfault

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#16660]
Steps to reproduce:

Download https://gcsweb.istio.io/gcs/istio-release-pipeline-data/daily-build/master-20190828-09-16/ and https://gcsweb.istio.io/gcs/istio-release-pipeline-data/daily-build/master-20190829-09-15/

```
$ ./istio-master-20190829-09-15/bin/istioctl
zsh: segmentation fault  ./istio-master-20190829-09-15/bin/istioctl
$ ./istio-master-20190828-09-16/bin/istioctl
Istio configuration command line utility for service operators to
debug and diagnose their Istio mesh.

Usage:
  istioctl [command]
...
```

So it broke on 8/29

Note this only seems to happen on release builds, if I build myself locally it works fine

**Comment by howardjohn** [comment]
fyi @sdake

**Comment by howardjohn** [comment]
The only plausible commit is https://github.com/istio/istio/commit/0200fdd0f05056db1092df8b491756934717d8d8

**Comment by mandarjog** [comment]
@fpesce did your investigation conclude here>

**Comment by rlenglet** [comment]
This is fixed in `release-1.3` by https://github.com/istio/istio/pull/16714.
This still needs to be fixed in `master`.

**Comment by geeknoid** [comment]
@ayj found the problem and it is:

https://github.com/istio/istio/blob/f6002411cc042232d0b69e7aeac1f5585ed8eea3/release/gcb/gcb_lib.sh#L180-L184

We're running SED over our binaries and corrupting them along the way...

**Comment by rlenglet** [comment]
@ayj are you going to submit a PR?

**Comment by rlenglet** [comment]
This will be fixed after the fix PR is backported into `release-1.3`.
